app/services/katello/pulp3/content_view_version/importable_repositories.rb

```diff
@@ -2,20 +2,25 @@ module Katello
   module Pulp3
     module ContentViewVersion
       class ImportableRepositories
```

Comment:
Unit tests for this similar to https://github.com/Katello/katello/blob/master/test/services/katello/pulp3/content_view_version/importable_repositories_test.rb#L9 for rh repositories

Reply:
Added